packages/icon/README.md

```diff
@@ -0,0 +1,56 @@
+# Icon
+
+Mineral UI `Icon` component. Find out more about [Mineral UI](https://github.com/mineral-ui/mineral-ui).
+
+This package provides a large number of prepackaged icon components as well as
+a generic Icon component to which you can provide your own SVG.
+
+
+## Installation
+
+1. Install project [peer dependencies](../../docs/peer-dependencies.md), if not already installed
+
+2. Install the package
+
+  ```sh
+  npm install --save @mineral-ui/icon
+  ```
+
+
+## Usage
+
+### Prepackaged Icons
+
+```jsx
+import React from 'react';
+import IconHelp from '@mineral-ui/icon/components/IconHelp';
```

Comment:
Changing `components` to `lib` for now.  Changing to root of `dist` is too much of a PITA for MVP.  Import path will look like `Import IconHelp from ‘@mineral-ui/icon/lib/IconHelp’`